x/osmolbp/client/cli/parse.go

```diff
@@ -0,0 +1,53 @@
+package cli
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/pflag"
+)
+
+type CreateLBPInputs createLBPInputs
+
+type XCreateLBPInputsExceptions struct {
+	CreateLBPInputs
+	Other *string // Other won't raise an error
+}
```

Comment:
Why do we need this wrapper type? Why do we need `Other *sting`?

If we really need it then it should be a not exported type and we should remove X from the prefix

Reply:
yeah we can directly use `CreateLBPInputs` type, will remove this